pages/community.html

```diff
@@ -2,43 +2,51 @@
 <#include "incl/header.html">
 
 <div class="framework">
-<div class="frameworklabel">Questions and discussion</div>
+<div class="frameworklabel">Communication channels</div>
 <div class="frameworkcontent">
 
 <p>
-Get in touch with the user community and ask questions about Coq on
-our <a href="https://coq.discourse.group">Discourse forum</a>. Posts
-in other languages than English are explicitly welcome there. There is
-also a historic mailing list called the
-<a href="https://sympa.inria.fr/sympa/arc/coq-club">Coq-Club</a>
-which has lots of subscribers.
+There are several channels to reach the user community and the
+development team:
 </p>
 
+<ul>
+<li>Our <a href="https://coq.zulipchat.com">Zulip chat</a>, for casual
+and high traffic discussions.
+<li>Our <a href="https://coq.discourse.group">Discourse forum</a>, for
+more structured and easily browsable discussions and Q&A.  Posts in
+other languages than English are explicitly welcome there.
+<li>Our historical mailing list, the
+<a href="https://sympa.inria.fr/sympa/arc/coq-club">Coq-Club</a>.
+</ul>
+
 <p>
-In addition, you may also ask questions about Coq on Stack Overflow (use
-the tag <a href="https://stackoverflow.com/questions/tagged/coq">coq</a>)
-or on the meta-theory of Coq on the
-<a href="https://cstheory.stackexchange.com">TCS Stack Exchange</a> (which also
-has a <a href="https://cstheory.stackexchange.com/questions/tagged/coq">coq</a>
-tag).
+The following channels have also a strong presence of Coq users:
```

Comment:
I believe it's more idiomatic to write
```suggestion
The following channels also have a strong presence of Coq users:
```